lib/activerecord-mysql-structure/active_record/structure_sql_sanitizer.rb

```diff
@@ -1,23 +1,89 @@
+# frozen_string_literal: true
+
 module ActiveRecordMySqlStructure
+  # This module is used by the enhanced rake task to sanitize the structure.sql
   class StructureSqlSanitizer
+    COLUMN_REGEX = /^\s*`\w+`.*,+$/
+
+    # When set to true, the sanitizer will sort the columns of each table. This
+    # will help keep a consistent column order in the outputted structure file.
+    # The columns will be sorted alphabetically.
+    def self.sorted_columns=(val)
+      @sorted_columns = val
+    end
+
+    # Returns whether the structure should have alphabetically sorted columns.
+    def self.sorted_columns?
+      @sorted_columns
+    end
+
     def self.sanitize(filename)
-      structure_lines = ''
+      new(filename).sanitize!
```

Comment:
Optional. Seems like also having `@sorted_columns` as ivar would simplify unit testing and it won't leak state if we use `new` directly:

```ruby
new(filename, sorted_columns: sorted_columns?).sanitize!
```

But currently it might not worth be worth the effort.

Reply:
That would def be easier to test. Will make the changes.